lumapps/api/client.py

```diff
@@ -1462,7 +1462,10 @@ def get_metadata_by_name(
             "metadata/list", instance=inst_id, parent=parent_id, familyId=parent_id
         )
         for metadata in metadatas:
-            if metadata["name"][self.first_lang].lower() == name.lower():
+            md_name = metadata["name"].get(self.first_lang)
+            if not md_name:
+                md_name = list(metadata["name"].values())[0]
+            if md_name.lower() == name.lower():
```

Comment:
We can try to match with all langs:
md_names = list(v.lower() for v in metadata['name'].values())

if name.lower() in md_names:
   ....

Reply:
Indeed, I'll make it so.